tool-openssl/x509.cc

```diff
@@ -18,6 +18,7 @@ static const argument_t kArguments[] = {
   { "-modulus", kBooleanArgument, "Prints out value of the modulus of the public key contained in the certificate" },
   { "-checkend", kOptionalArgument, "Check whether cert expires in the next arg seconds" },
   { "-days", kOptionalArgument, "Number of days until newly generated certificate expires - default 30" },
+  {"-text", kBooleanArgument, "Pretty print the contents of the certificate"},
```

Comment:
NP: The other lines have a space after the opening bracket: `{ "-text", ...`

Reply:
related to NP: I advocate for running clang-format on the files we're touching while we're at it.